skyportal/app_server.py

```diff
@@ -18,6 +18,7 @@
     FacilityMessageHandler,
     GroupHandler,
     GroupUserHandler,
+    GroupUsersFromOtherGroupsHandler,
```

Comment:
How about `MergeGroupsHandler`?

Reply:
It doesn't merge them though. They're still distinct and users added to any groups after the fact aren't added to the target group. That'd be a misleading name